assets/app/view/game/buy_trains.rb

```diff
@@ -302,7 +302,7 @@ def from_depot(depot_trains, corporation)
             president_assist, _fee = @game.president_assisted_buy(@corporation, train, price)
             entity = @corporation
 
-            if @selected_company&.owner == @corporation
+            if [@corporation, @corporation.owner].include?(@selected_company&.owner)
```

Comment:
Will this break other games where the corp owner can't activate the power? I wonder if you need to check who is allowed to play the ability?

Reply:
if the ability is passive, that it shouldn't have an impact.
The game will need to enable the ability to show up when owned by player, and only then will this code path execute.
Checking for active abilities

Reply:
doesn't break 18usa. Tries two other passive games.